### ISSUE TITLE
Floated label with icon and text - line breaks

### ISSUE THREAD

**Issue by brendon** [Semantic-Org/Semantic-UI#4217]
It seems that one can't currently have an icon and text label that is floated:

![screen shot 2016-07-05 at 9 15 25 pm](https://cloud.githubusercontent.com/assets/81871/16579838/ba842c92-42f5-11e6-8250-9755376ac26a.png)

My fix for this was to set:

``` css
.ui.floating.label {
  white-space: nowrap;
}
```

![screen shot 2016-07-05 at 9 18 22 pm](https://cloud.githubusercontent.com/assets/81871/16579909/15849ba4-42f6-11e6-8ea0-0dd6184b3dda.png)

The icon also probably has unnecessary right margin given it's inline with the text. The `fitted` option is overridden by the label container.

**Comment by pierre-weceipt** [comment]
Same  as #4155

**Comment by brendon** [comment]
Not quite. Yes the last part about the icon's margin is the same, but I'm arguing that we should prevent wrap for labels like this since they're only meant to be small, and that then means one can use icons in them.

**Comment by vinayakkulkarni** [comment]
You can use your workaround as [Jack declined the PR ](https://github.com/Semantic-Org/Semantic-UI/issues/4155#issuecomment-227926946)

**Comment by brendon** [comment]
As per my last comment, that PR refers to something else, so I don't think this is a duplicate.

The point of this issue is that one can't use an icon and text in a floated label because line-wrapping occurs (see the original message).

**Comment by Banandrew** [comment]
You’re right, sorry about that. Reopening as enhancement.

**Comment by brendon** [comment]
Thanks @Banandrew :)

**Comment by brendon** [comment]
Ping

**Comment by gustavo80br** [comment]
This is really a problem, labels should not break lines, specially if using images/icons

**Comment by pavangayakwad** [comment]
any updates on this?

**Comment by lubber-de** [comment]
@brendon Thanks. We adopted your fix in https://github.com/fomantic/Fomantic-UI/pull/404 and supposed to be part of upcoming Fomantic-UI 2.7.2

**Comment by brendon** [comment]
Thanks @lubber-de :D
